services-js/registry-certs/client/common/checkout/ReviewContent.tsx

```diff
@@ -426,7 +405,10 @@ export default class ReviewContent extends React.Component<Props, State> {
             />
             <span className="cb-l">
               I understand that{' '}
-              <strong>{certificatesName} are non-refundable</strong>.
+              <strong>
+                {this.props.certificateType} certificates are non-refundable
```

Comment:
Same note here, where you're mixing in a symbol (certificateType) with human-readable text